lib/scrabble.rb

```diff
@@ -10,4 +10,59 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+
+
+    if word == nil
+      return nil
+    else
+      letters = word.chars
+
+      return_score = letters.map do |score|
+        score_it = score.upcase
+        point_values[score_it]
+      end
+    end
+
+    final_score = return_score.sum
```

Comment:
Great use of sum!  You don't _need_ to set this to a variable and return the variable, you could just have `return_score.sum` as your last line of the method.